Add blogpost for forthcoming Packit 1.0.0 (#978)
packit/packit.dev: posts/packit_1_0_0_action_required/index.md

---
title: "Packit 1.0.0 countdown - action may be required"
date: 2024-12-09T12:00:00+00:00
authors: mmassari
tags:
  - configuration
---

After 105 minor releases we are almost ready to announce our first major release, Packit 1.0.0!

<!--truncate-->

## What's New

> We will **remove all since long deprecated keys from the configuration file**.

Reviewer: ```suggestion
We will **remove all long-deprecated keys from the configuration file**.
```

sounds a bit more natural to me

> `packit-service` jobs are tight with `packit CLI` so be aware that for those projects still using deprecated keys in their `packit.yaml` project file we have created a migration pull request.

Reviewer 2: ```suggestion
`packit-service` jobs are tied with `packit CLI` so be aware that for those projects still using the deprecated keys in their `packit.yaml` configuration file we have created a migration pull request.
```

> `packit-service` jobs are tight with `packit CLI` so be aware that for those projects still using deprecated keys in their `packit.yaml` project file we have created a migration pull request.

Reviewer: I would maybe rephrase the beginning a bit, telling that the code regarding the configuration is placed in the `packit` package, therefore `packit-service` users will be affected as well.

> Please, **check if you have any new PR for your project created by us, review and merge it**. Otherwise, you might experience issues with the `packit-service` jobs once the new packit version is released.

Reviewer: ```suggestion
Please, **check if you have any new PR for your project created by us, review and merge it**. Otherwise, you might experience issues with the `packit-service` jobs once the Packit 1.0.0 is released.
```

I would also link the user that will create the PRs in the [by us]

Author: I am using the packit-service tokens, see [here](https://gitlab.com/packit-service/hello-world/-/merge_requests/1298). So the user is Packit Service. Should I use different tokens? Probably I will need to create them for a different user in all gitlab.org instances...

Reviewer: great, no, I think using the Packit Service user makes sense, so maybe just mention that explicitly (`created by the packit-service bot account`)

> We plan to release Packit 1.0.0 on **January 2025**.

Reviewer 2: ```suggestion
We plan to release Packit 1.0.0 in **January 2025**.
```

### List of removed keys

> - Job `build` does not exist anymore, changed in `copr_build`.

Reviewer: ```suggestion
- Job `build` does not exist anymore, changed to `copr_build`.
```

---
title: "Packit 1.0.0 countdown - action may be required"
date: 2024-12-09T12:00:00+00:00
authors: mmassari
tags:
  - configuration
---

After 105 minor releases we are almost ready to announce our first major release, Packit 1.0.0!

<!--truncate-->

## What's New

We will **remove all long-deprecated keys from the configuration file**.

`packit-service` jobs are tied with `packit CLI`. The _packit configuration code_ is defined in the `packit` package and used by the `packit-service`.
So be aware that for those projects still using the deprecated keys in their `packit.yaml` configuration file we have created a migration pull request.

Please, **check if you have any new PR for your project created by the packit-service bot account, review and merge it**. Otherwise, you might experience issues with the `packit-service` jobs once the new packit version is released.

We plan to release Packit 1.0.0 in **January 2025**.

### List of removed keys

- Job type `build` has been changed to `copr_build`.

> - Job type `production_build` has been changed to `upstream_koji_build`.

Reviewer 3: We can mention that this is for explicitness after introducing more build types.

- Key `upstream_project_name` has been changed to `upstream_package_name`.

> - Key `synced_files` has been changed to `files_to_sync`.

Reviewer 3: Do we want to mention a difference in behaviour? Or, at least link docs?

Author: I will link the docs
